Linux/Car_Client/chronoscomm.cpp

```diff
@@ -131,6 +134,8 @@ void ChronosComm::closeConnection()
     mTcpSocket->close();
     mUdpSocket->close();
     mTcpState = 0;
+    mGpioControl->GPIO_Write(4, 0);
+    mGpioControl->unsetGPIO(4);
```

Comment:
We should do this also when going from RUNNING (or the RC equivalent) to CONNECTED i.e. when finishing a run without disconnecting.

Reply:
Will look into this. I dont think the RC car has any state handling other than some variables like  mIsArmed = false;  mIsStarted = false; etc